crates/re_data_ui/src/app_id.rs

```diff
@@ -0,0 +1,54 @@
+use re_entity_db::EntityDb;
+use re_log_types::ApplicationId;
+use re_viewer_context::{UiVerbosity, ViewerContext};
+
+use crate::item_ui::entity_db_button_ui;
+
+impl crate::DataUi for ApplicationId {
+    fn data_ui(
+        &self,
+        ctx: &ViewerContext<'_>,
+        ui: &mut egui::Ui,
+        verbosity: UiVerbosity,
+        _query: &re_data_store::LatestAtQuery,
+        _store: &re_data_store::DataStore,
+    ) {
+        egui::Grid::new("application_id")
+            .num_columns(2)
+            .show(ui, |ui| {
+                ui.label("Application ID");
+                ui.label(self.to_string());
+                if self == &ctx.store_context.app_id {
+                    ui.label("(active)");
+                }
+                ui.end_row();
+            });
+
+        if verbosity == UiVerbosity::Small {
+            return;
+        }
+
+        // Find all recordings with this app id
+        let recordings: Vec<&EntityDb> = ctx
```

Comment:
Would be nice to have the blueprints here as well.

Reply:
Maybe, but could also be very confusing. Do we show all the blueprints? Or just the active and default ones? What if the active and default are the same? Etc. I opted to keep it simple for now.